Ascend.java

```diff
@@ -1,7 +1,19 @@
-// Generates three integer random numbers in a given range,

```

Comment:
OVERALL: 
1. good submission, there is a slight mistake in Bill3.java which deducted you points, I recommend reading the question first 
2. Please review TimeFormat.java the code there is very clustered. 
3. Few Styling mistakes.
4. Write better commit names